lforms-converter.js

```diff
@@ -398,6 +398,9 @@ function doSkipLogic(root) {
 
   traverseItems(root, function(item, ancestors) {
     if(item.skipLogic) {
+      if (item.skipLogic.condition != undefined && item.skipLogic.condition.trim() === "") {
+        delete item.skipLogic;
+      } else {
       // This is target item. Parse 'condition' to look for source item
       var tokens = item.skipLogic.condition.split('=');
```

Comment:
Peter, can we send Ajay an example of a form from prod that will have skip logic that breaks lf-converter.
I think anything with more than one AND or anything where the logic is broken. Or date > 1998, stuff like that.